### ISSUE TITLE
undefined is add at url end when refresh page

### ISSUE THREAD

**Issue by Billybob** [jsbeckr/django-inertia-example#1]
Hi,
http://127.0.0.1:8000 -> http://127.0.0.1:8000/undefined
http://127.0.0.1:8000/dummy -> http://127.0.0.1:8000/dummy/undefined
When refresh page.

`<inertia-link>` is ok

any idea?

**Comment by Billybob** [comment]
I add **url** in **_build_context** (inertia-django) and is ok!!!

**Comment by jsbeckr** [comment]
I wrote this adapter for an old version of inertia. Did you update to the latest version? I'm pretty sure they made some changes that makes the adapter not compatible to the newest version.

**Comment by Billybob** [comment]
yes, latest version of inertia and inertia-vue! After a few changes it works.

```
import { InertiaApp } from "@inertiajs/inertia-vue";
...
Vue.use(InertiaApp)
...
new Vue({
  render: h => h(InertiaApp, {
      props: {
        ...
      },
    }),
}).$mount(app)
```


An update is planned?

**Comment by jsbeckr** [comment]
No unfortunately not. I don't use inertia at the moment. Do you want to take over maintenance?

**Comment by Billybob** [comment]
Not very comfortable with JavaScript but I want to try!
